mate_rov.html

```diff
@@ -222,27 +222,26 @@ <h2 class="title">About MATE ROV</h2>
-					 change raising ocean temperatures, affecting the health of coral reefs. Global Community - and the requests for proposals is simple:
-					  design and build a remotely operated vehicle and the necessary sensors and tooling to tackle the realworld problems of plastics in our
-					   ocean, climate change's impact on coral reefs, and the consequences of poor environmental practices on our inland waterways.
-						The solution to these problems focuses on remediation meaning cleaning up rather than fixing the root of these problems.</p>
-
+						<p class="about-des">
+							As founding members, we wanted to introduce our college undergraduate students to underwater technology. For this, 
+							Marine Technology Society was the best choice as their vision aligned with ours, which is to spread awareness, understanding, and 
+							the advancement and application of marine technology. We wanted to allow our students to have a platform by which they could interact
+							with other international students through their research and innovative ideas. The eight of us decided to have a chapter of MTS at the 
+							college after knowing the countless yet priceless opportunities the students can get by being a member of the MTS chapter in research and innovation of underwater technology.
+						</p>
 						<p class="about-des">The biggest challenge for community is adapting to a new environment that favors smarter, cleaner, and healthier
-							 products and services and moving away from the mindset of the industrial era when pollution was tolerated, labor was just another
-							  cost ( and not real people), and bigger was better (and meant more money). Getting everyone thinking the same way and viewing the
-							   world as <strong>"What we have out there is all we have; there is no more"</strong>.</p>
-
-							   <p class="about-des">And this is where your mission begins.
-								   <br>
-									Task1: The Ubiquitous Problem of Plastic Pollution
-									<br>
-									Task2: The Catastrophic Impact of Climate Change on Coral Reefs
-									<br>
-									Task 3: Maintaining Healthy Waterways Part II: Delaware River and Bay
-									<br><br>
-							   </p>	   
+							products and services and moving away from the mindset of the industrial era when pollution was tolerated, labor was just another
+							 cost ( and not real people), and bigger was better (and meant more money). Getting everyone thinking the same way and viewing the
+							  world as <strong>"What we have out there is all we have; there is no more"</strong>.</p>
+						<p class="about-des">And this is where your mission begins.
+							<br>
+							Task1: The Ubiquitous Problem of Plastic Pollution
+							<br>
+							Task2: The Catastrophic Impact of Climate Change on Coral Reefs
+							<br>
+							Task 3: Maintaining Healthy Waterways Part II: Delaware River and Bay
+							<br><br>
+						</p>	   
```

Comment:
Remove this also.
And change the file name from `mate_rov.html` to `mts_auvzhcet.html`

Reply:
done